optimism/client.go

```diff
@@ -915,7 +916,30 @@ func traceOps(block *types.Block, calls []*flatCall, startIndex int) []*RosettaT
 				},
 				Metadata: metadata,
 			}
-			ops = append(ops, burnOp)
+			ops = append(ops, burnDebitOp)
+			
+			lastOpIndex := ops[len(ops)-1].OperationIdentifier.Index
+			burnCreditOp := &RosettaTypes.Operation{
+				OperationIdentifier: &RosettaTypes.OperationIdentifier{
+					Index: int64(len(ops) + startIndex),
+				},
+				RelatedOperations: []*RosettaTypes.OperationIdentifier{
+					{
+						Index: lastOpIndex,
+					},
+				},
+				Type:   trace.Type,
+				Status: RosettaTypes.String(opStatus),
+				Account: &RosettaTypes.AccountIdentifier{
+					Address: zeroAddr,
```

Comment:
Does the Rosetta data validation check account for zero addresses? I just realized that this might break it, but perhaps validation could be configured to ignore 0x0.

Reply:
Great question, I've floated it along to the team. @jacquescaocb @GeekArthur happen to have an idea?

Reply:
rosetta checks all accounts appeared in block tx, unless it is exempted.

Reply:
I dont think we need to validate zero address, we should mark it.

Reply:
@Inphi @andrewkmin We validated zero address, IIRC that's why we have this commit https://github.com/Inphi/optimism-rosetta/commit/bff0f67fd5c53ad58e12ddc78f9d1d8582ef46bb after we detected reconciliation issue on zero address. If really don't want to validate it we can put it in `exempt_accounts` field under `config.json`, but I am not sure if ignoring it can expose security vulnerability such as double mint issue

Reply:
I've quickly run rosetta validation on a few blocks that include a burn tx and it is indeed failing. We could add the zero address to the exempt_accounts in the configs, but could potentially run into security issues. Do you have any thoughts on that @Inphi ?

Reply:
This is OK as long as the counterparty to a zero address transaction is still accounted for.